example/usps_api/config/config_parser.h

```diff
@@ -1,3 +1,16 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
 #include "json/json.h"
 namespace Config {
```

Comment:
Rather than a Config namespace, it might be better to put the entire project in its own namespace, https://google.github.io/styleguide/cppguide.html#Namespace_Names.

Reply:
Done